tests/conftest.py

```diff
@@ -71,6 +71,7 @@
 AIRFLOW_TESTS_DIR = Path(os.path.dirname(os.path.realpath(__file__))).resolve()
 AIRFLOW_SOURCES_ROOT_DIR = AIRFLOW_TESTS_DIR.parent.parent
 
+os.environ["AIRFLOW__CORE__PLUGINS_FOLDER"] = os.fspath(AIRFLOW_TESTS_DIR / "plugins")
```

Comment:
This removes a side-effect of change that was implemented in BREEZE where `/files/plugins` is set as default when you enter BREEZE. Some tests were failing expecting the plugins folder to point to internal plugins, so I added it in conftest.py to have it working the same regardless if you are in Breeze or not.